torch/autograd/_functions/pointwise.py

```diff
@@ -61,22 +62,28 @@ def backward(self, grad_output):
 
 class Sigmoid(InplaceFunction):
 
-    def forward(self, i):
-        if self.inplace:
-            self.mark_dirty(i)
+    @staticmethod
+    def forward(ctx, i, inplace=False):
+        ctx.input_size = i.size()
```

Comment:
Sorry, I missed this. It's unnecessary, you can just use `grad_output.size()` in `backward`.